tutorials/shaders/shader_reference/spatial_shader.rst

```diff
@@ -427,73 +428,73 @@ If you want the lights to add together, add the light contribution to ``DIFFUSE_
+| in vec4 **FRAGCOORD**             | Coordinate of pixel center in screen space.                            |
+|                                   | ``xy`` specifies position in window, ``z``                             |
+|                                   | specifies fragment depth if ``DEPTH`` is not used.                     |
+|                                   | Origin is lower-left.                                                  |
++-----------------------------------+------------------------------------------------------------------------+
+| in mat4 **MODEL_MATRIX**          | Model/local space to world space transform.                            |
++-----------------------------------+------------------------------------------------------------------------+
+| in mat4 **INV_VIEW_MATRIX**       | View space to world space transform.                                   |
++-----------------------------------+------------------------------------------------------------------------+
+| in mat4 **VIEW_MATRIX**           | World space to view space transform.                                   |
++-----------------------------------+------------------------------------------------------------------------+
+| in mat4 **PROJECTION_MATRIX**     | View space to clip space transform.                                    |
++-----------------------------------+------------------------------------------------------------------------+
+| in mat4 **INV_PROJECTION_MATRIX** | Clip space to view space transform.                                    |
++-----------------------------------+------------------------------------------------------------------------+
+| in vec3 **NORMAL**                | Normal vector, in view space.                                          |
++-----------------------------------+------------------------------------------------------------------------+
+| in vec2 **UV**                    | UV that comes from the ``vertex()`` function.                          |
++-----------------------------------+------------------------------------------------------------------------+
+| in vec2 **UV2**                   | UV2 that comes from the ``vertex()`` function.                         |
++-----------------------------------+------------------------------------------------------------------------+
+| in vec3 **VIEW**                  | View vector, in view space.                                            |
++-----------------------------------+------------------------------------------------------------------------+
+| in vec3 **LIGHT**                 | Light vector, in view space.                                           |
++-----------------------------------+------------------------------------------------------------------------+
+| in vec3 **LIGHT_COLOR**           | :ref:`Light color<class_Light3D_property_light_color>` multiplied by   |
```

Comment:
These two were awkward to write because you can't have links in a code block. I think I found a decent compromise.
![firefox_fSIRLFZFLl](https://github.com/user-attachments/assets/039c2308-999c-4a46-b528-471c1691f0c1)